docs/manual/search.xml

```diff
@@ -0,0 +1,8 @@
+<OpenSearchDescription xmlns="http://a9.com/-/spec/opensearch/1.1/"
+                       xmlns:moz="http://www.mozilla.org/2006/browser/search/">
+    <ShortName>Contao Manual</ShortName>
+    <Description>Contao Manual</Description>
+    <InputEncoding>UTF-8</InputEncoding>
+    <Image width="16" height="16" type="image/x-icon">https://contao.org/favicon.ico</Image>
+    <Url type="text/html" method="get" template="https://docs.contao.org/manual/?search={searchTerms}"/>
```

Comment:
Will this work? The manual is separated into `en/` and `de/`. Currently `https://docs.contao.org/manual/` just redirects to `https://docs.contao.org/manual/de` via HTML (will be changed in the future).

Reply:
No, then it won't work. Any idea how to solve it? It's possible to keep the param when doing the redirect or do we need to ship different search.xml files.

Reply:
Yes, we'll need different `search.xml` files. Hm, what happens if you create a `docs/manual/de/search.xml` manually, will it be copied over when you run Hugo?

Reply:
I figured out how to solve it. I use a custom output format for the home page now which creates the `search.xml`.